string id
float32[3] pose
float32[3] min
float32[3] max
float32[1] orientation
float32[1] velocity